hazelcast/src/main/java/com/hazelcast/core/IMap.java

```diff
@@ -73,6 +74,25 @@
  * </p>
  * <p>This class does <em>not</em> allow <tt>null</tt> to be used as a key or value.</p>
  *
+ * <p>Entry Processing</p>
+ * The following operations are lock-aware, since they operate on a single key only.
+ * If the key is locked the EntryProcessor will run until it acquires the lock.
+ * <ul>
+ * <li>{ @link IMap#executeOnKey(Object, EntryProcessor) }</li>
+ * <li>{ @link IMap#submitToKey(Object, EntryProcessor) }</li>
+ * <li>{ @link IMap#submitToKey(Object, EntryProcessor, ExecutionCallback) }</li>
+ * </ul>
+ * There are however following methods that run the EntryProcessor on more than one entry. These operations are not lock-aware.
+ * The EntryProcessor will process the entries no matter if they are locked or not.
+ * The user may however check if an entry is locked by casting the {@link java.util.Map.Entry} to ]
+ * {@link LockAware} and invoking the {@link LockAware#isLocked()} method.
+ * <ul>
+ * <li>{ @link IMap#executeOnEntries(EntryProcessor) }</li>
+ * <li>{ @link IMap#executeOnEntries(EntryProcessor, Predicate) } </li>
+ * <li>{ @link IMap#executeOnKeys(Set, EntryProcessor) }</li>
```

Comment:
Remove the spaces after `{` and before `}` for it to become links. We could also add some javadoc for the lock awareness on those methods.